workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/fragments/group/liferay-learn/fragments/public-sites-navigation/index.js

```diff
@@ -313,4 +313,42 @@ window.addEventListener('keyup', (event) => {
 
 		siteSearchWrapper.classList.add('search-open');
 	}
-});
\ No newline at end of file
+});
+
+const capabilitiesLinkContainer = document.getElementsByClassName(
```

Comment:
@rtienpal I am uncertain what we are trying to achieve with this code. What is the goal of the javascript and what part of the ticket requires it?